trail/trail/templates/about.html

```diff
@@ -3,18 +3,103 @@
 
 {% block about-nav-state%}active{% endblock %}
 
-<!-- There is no social footer on this page so we can replace the whole thing with nothing.  -->
-{% block page_footer %} {% endblock %}
-
 {% block content %}
 <!-- Post -->
 <section class="post">
+		<!--
 		<header class="major">
-				<h1>This is<br />
-						all about me</h1>
+				<h1>About Trailblazer</h1>
 		</header>
+		-->
+
+		<h2>The problem &#128752;</h2>
```

Comment:
I think we should make all our titles anchor points too. Something like:
```
<h2>
  <a name="our-approach"></a>
  <a class="headerlink" href="#our-approach" title="Permalink to this headline">Our approach &#129321;</a>
</h2>
```

Then people could link directly to a section like `/about/#our-approach` which I just found always very practical.

Reply:
I knew that was a thing, but wasn't sure how - thanks